pkg/actors/actors.go

```diff
@@ -535,18 +541,45 @@ func (a *actorsRuntime) connectToPlacementService(placementAddress, hostAddress
 	}()
 }
 
+func (a *actorsRuntime) getClientOptions(certChain *dapr_credentials.CertChain) ([]grpc.DialOption, error) {
```

Comment:
can this also be moved to credentials/grpc.go

Reply:
done!